src/Broker/ActiveMq/ActiveMq.php

```diff
@@ -95,8 +95,13 @@ public function getAckFrame(Frame $frame, $transactionId = null)
     /**
      * @inheritdoc
      */
-    public function getNackFrame(Frame $frame, $transactionId = null)
+    public function getNackFrame(Frame $frame, $transactionId = null, $requeue = null)
     {
+        if ($requeue !== null) {
+            throw new \LogicException(
+                'requeue header not supported by ActiveMQ. Please read ActiveMQ DLQ documentation.'
```

Comment:
why do you mention the docs here? is ActiveMQ DLQ a euqivalent feature as $requeue?

Reply:
yes, though configured through activemq.xml, not something you can do individually with message headers.